Makefile

```diff
@@ -2,7 +2,15 @@
 
 all: deps build-client
 
-deps: clean deps-server optimize autoload deps-client
+deps: clean deps-server optimize autoload deps-client gems
+
+deps-production: clean deps-server optimize autoload deps-client gems-production
```

Comment:
Doesn't this line need `gems` before `gems-production`, in case it's run independently?